### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by flipactual** [incrediblesound/story-graph#5]
Would you like the codebase to be tested? If so, I'm happy to PR.

**Comment by incrediblesound** [comment]
Yes, that would be great! I originally made the code base more modular in anticipation of testing. There is one concern I have which is that the input and output types of the various functions are unclear. Story Graph is inherently dynamic and flexible because it is rendering stories, but that does not mean that we can't keep the various internal APIs reasonably consistent. 

It would be great if you would raise an issue if you find any functions where the inputs and outputs are so flexible that it is difficult to predict what they will be. Basically if you find anything confusing in the codebase let me know! We can see if improvements can be made and then testing will be easier.

**Comment by flipactual** [comment]
Do you have a preference for testing framework? I would use mocha and istanbul for coverage if you don't have a preference.

**Comment by incrediblesound** [comment]
No preference, Mocha and Istanbul would be fine.

**Comment by flipactual** [comment]
Will do!

In cases where the returned values are randomized, we can still test that at least the proper type is returned and that it contains one of the possible outputs.
